Documentation/ContentObjects/Extbaseplugin/_CodeSnippets/_libMyPlugin.typoscript

```diff
@@ -0,0 +1,6 @@
+lib.myPlugin = EXTBASEPLUGIN
+lib.myPlugin {
+    extensionName = MyExtension
+    pluginName = MyPlugIn1
+    settings.detailPid = 42
```

Comment:
Have you tested that? Can 't find "settings" as in official list of allowed properties for that cObj.

Reply:
Yes I tested it and it is working. Probably some Extbase magic...?